<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>man_hinh_dang_nhap</class>
 <widget class="QWidget" name="man_hinh_dang_nhap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_dang_nhap" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>300</width>
      <height>500</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/uit.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>170</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ĐĂNG NHẬP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tai_khoan">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>140</y>
      <width>210</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 0);
border: none;
border-bottom:2px solid rgba(46, 82, 101, 200);
color: rgba(0, 0, 0, 240);
padding-bottom: 7px;
</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Nhập Tài khoản</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mat_khau">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>210</y>
      <width>210</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 0);
border: none;
border-bottom:2px solid rgba(46, 82, 101, 200);
color: rgba(0, 0, 0, 240);
padding-bottom: 7px;
</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Nhập mật khẩu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dang_nhap">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>310</y>
      <width>210</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(46, 82, 101, 200);
border-color: rgba(0, 0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Đăng nhập</string>
    </property>
   </widget>
   <widget class="QPushButton" name="thoat">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>380</y>
      <width>210</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(46, 82, 101, 200);
border-color: rgba(0, 0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Thoát</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="hien_mat_khau">
    <property name="geometry">
     <rect>
      <x>537</x>
      <y>210</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
